.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
IDE settings ideally shouldn't be committed to source control FYI. Not part of the coursework, just a heads up!